addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -610,6 +610,68 @@ QUnit.test('chat window: open / close', async function (assert) {
     );
 });
 
+QUnit.test('chat window (mobile): open / close with no server notification', async function (assert) {
+    assert.expect(5);
+
+    // channel that is expected to be found in the messaging menu
+    // with random UUID, will be asserted during the test
```

Comment:
these comments were a bad idea (they look important but in practice it's a waste of time reading them), so don't copy them

Reply:
Done